app/src/main/java/at/aau/serg/websocketdemoapp/activities/ActiveGame.java

```diff
@@ -315,6 +319,21 @@ public void tableButtonClicked() {
         startActivity(intent);
     }
 
+    private void subscribeForGameEndEvent() {
+        stompHandler.subscribeForGameEnd(dataHandler.getLobbyCode(), this::handleGameEnd);
+    }
+
+    private void handleGameEnd(String response) {
+        if (response.equals("Game ended")) {
```

Comment:
Etwas seltsam da genau auf diesen String zu überprüfen -> aber passt

Reply:
Theoretisch für Error Handling aber stimmt, in diesem Fall unnötig. Wird entfernt